components/mega-menu-dropdown/index.test.ts

```diff
@@ -22,4 +22,61 @@ describe('mega menu dropdown', () => {
     );
     await expect(el).to.be.accessible();
   });
+  it('can be opened and closed', async () => {
+    const el: HTMLElement = await fixture(
+      html`<qiskit-mega-menu-dropdown
+        placeholder="Browse all content"
+        .content="${tripleColumnMultiBlock}"
+      ></qiskit-mega-menu-dropdown>`
+    );
+
+    if (!el.shadowRoot) {
+      expect.fail('Cannot find shadow root');
+    }
+
+    const contentSelector = '.content';
+    const filterButton: HTMLButtonElement | null =
+      el.shadowRoot.querySelector('.filter__button');
+
+    expect(el.shadowRoot?.querySelector(contentSelector)).to.not.exist;
+    expect(filterButton).to.exist;
+
+    filterButton?.click();
+    await nextFrame();
+    expect(filterButton).to.be.visible;
+    expect(el.shadowRoot?.querySelector(contentSelector)).to.exist.and.be
+      .visible;
+
+    filterButton?.click();
+    await nextFrame();
+    expect(filterButton).to.be.visible;
+    expect(el.shadowRoot?.querySelector(contentSelector)).to.not.exist;
+  });
+
+  it('should show empty view', async () => {
+    const el: HTMLElement = await fixture(
```

Comment:
question: is it possible to define this once, above all tests, and refer to them within each?